### PR TITLE
Improve the Marathon LB article

### PR DESCRIPTION
**Pull request by brouberol**
Signed-off-by: Balthazar Rouberol <balthazar.rouberol@corp.ovh.com>

### CODE REVIEW

File: en/_posts/2016-04-22-marathon-load-balancer.md [hunk 1]
```diff
@@ -6,18 +6,73 @@ author: devatoria
 lang: en
 ---
 
-The Marathon load balancer can be configured to use a specific domain name, to make you application URL more user-friendly. For example, let's say we have a nginx application and we want it to be available at `https://myapp.info`.
+In this tutorial, we'll see how to configure TODO [HAProxy](https://cbonte.github.io/haproxy-dconv/intro-1.6.html) load balancer through application labels.
```
Comment (yadutaf) [inline en/_posts/2016-04-22-marathon-load-balancer.md]: TODO ?

File: en/_posts/2016-04-22-marathon-load-balancer.md [hunk 2]
```diff
@@ -6,18 +6,73 @@ author: devatoria
...
 
-Considering your domain name is pointing to your load balancer URL (https://<username>.lb.sbg-1.containers.ovh.net), you just have to add the following labels:
+## First and foremost
+
+This tutorial assumes you have an nginx container up and running. If that's not the case, please read [the previous tutorial](/kb/en/docker/quick-start-with-marathon.html).
+
+
+## Architecture
+
+Inside the `[marathon-lb](https://github.com/mesosphere/marathon-lb` container, the marathon-lb script marathon_lb.py connects to the marathon API to retrieve all running apps, generates a HAProxy config and reloads HAProxy. By default, marathon-lb binds to the service port of every application and sends incoming requests to the application instances.
```
Comment (yadutaf) [inline en/_posts/2016-04-22-marathon-load-balancer.md]: Maybe capitalize the 'M' or Marathon?

File: en/_posts/2016-04-22-marathon-load-balancer.md (hunk 1 above)
Comment (brouberol) [inline en/_posts/2016-04-22-marathon-load-balancer.md]: Oops! Well spotted. I struggled to find the appropriate wording, and forgot about it.

**Comment by yadutaf** [comment]
Looks good :)

**Pull request merged**